template/html/common/layout.html

```diff
@@ -35,6 +35,7 @@
 
     <meta name="theme-color" content="{{ theme_color .theme }}">
     <link rel="stylesheet" type="text/css" href="{{ route "stylesheet" "name" .theme }}?{{ .theme_checksum }}">
+    <link rel="stylesheet" type="text/css" href="{{ route "stylesheet" "name" "custom_css" }}">
```

Comment:
This is going to generate an HTTP request for users without custom style sheet. Perhaps, you should consider using inline styles instead: `<style>...</style>`.

Reply:
I tried using inline styles but it gave me an inline style content security policy, so i went with this way, should i change the headers from the server ot allow it?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/default-src